src/Catalog/Helpers/FeedHelpers.cs

```diff
@@ -261,10 +262,22 @@ private static DateTime ForceUtc(DateTime date)
                 return lastDate;
             }
 
+            var packageProcessingDurationTelemetryList = new List<DurationMetric>();
```

Comment:
Here I measure the processing time of a package from being read from the feed to written to catalog. 
It's a bit tricky since they are read one by one, by commited in batch, hence the list of DuartionMetric that need to be disposed together. If we see this pattern in other places in the code I suggest having a class to manage the list creation and dispose... :)